src/couch_mrview/src/couch_mrview_test_util.erl

```diff
@@ -53,7 +53,8 @@ ddoc(changes) ->
     couch_doc:from_json_obj({[
         {<<"_id">>, <<"_design/bar">>},
         {<<"options">>, {[
-            {<<"seq_indexed">>, true}
+            {<<"seq_indexed">>, true},
+            {<<"keyseq_indexed">>, true}
```

Comment:
If query only contains "since", it should be enough to enable "seq_indexed". However, in some of test cases in this test suite, the query contains both "start_key" and "end_key".  This needs to enable "keyseq_indexed". Thus, the test can check `key_byseq_btree` instead of `seq_btree` to get expected results. I sorted out the test cases and only enable these two options when necessary.